.github/workflows/docker-build.yml

```
@@ -0,0 +1,125 @@
+name: docker-build
+
+workflow_call:
+
+env:
+  REGISTRY_IMAGE: fcsproj/fastcalosim
+
+jobs:
+
+  # Step 2: Run Docker build ONLY if relevant files changed and not a closed PR
+  build:
```

actionlint (1.7.4)

job "build" needs job "check-files" which does not exist in this workflow

(job-needs)